### PR TITLE
Added log message when Video.Placement property is not available

### PR DESCRIPTION
**Pull request by kapil-tuptewar**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

- test parameters for validating bids
```
{
  bidder: '<bidder name>',
  params: {
    // ...
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
- [ ] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

File: modules/pubmaticBidAdapter.js [hunk 1]
```diff
@@ -537,12 +538,21 @@ function _createBannerRequest(bid) {
   return bannerObj;
 }
 
+export function checkVideoPlacement(videoData) {
+  // Checking if placement property send to video config also
+  // Checking if property is not empty.
+  if (!videoData.hasOwnProperty('placement') || !videoData.placement) {
```
Comment (pm-harshad-mane) [inline modules/pubmaticBidAdapter.js:544]: you can use utils.deepAccess to retrieve videoData.placement w/o adding checks, the util function will return undefined if the property in the access chain is not present

File: modules/pubmaticBidAdapter.js [hunk 2]
```diff
@@ -18,6 +18,7 @@ const DEFAULT_HEIGHT = 0;
 const PREBID_NATIVE_HELP_LINK = 'http://prebid.org/dev-docs/show-native-ads.html';
 const PUBLICATION = 'pubmatic'; // Your publication on Blue Billywig, potentially with environment (e.g. publication.bbvms.com or publication.test.bbvms.com)
 const RENDERER_URL = 'https://pubmatic.bbvms.com/r/'.concat('$RENDERER', '.js'); // URL of the renderer application
+const VIDEO_PLACEMENT_MISSING_MSG = 'Video.Placement param missing';
```
Comment (pm-harshad-mane) [inline modules/pubmaticBidAdapter.js:21]: naming message related constants with "MSG_" at start will help in organizing code.

File: modules/pubmaticBidAdapter.js (hunk 1 above)
Comment (kapil-tuptewar) [reply modules/pubmaticBidAdapter.js]: Replaced above code with utils.deepAccess method

File: modules/pubmaticBidAdapter.js (hunk 2 above)
Comment (kapil-tuptewar) [reply modules/pubmaticBidAdapter.js]: Renamed above variable with "MSG_" at the start.

**Pull request merged**